### DISCUSSION TITLE
The function Perplexity Search Manifold Pipe does not work, shows "Error: Failed to get socket ID"

### DISCUSSION THREAD

**Discussion by foxworld306** [open-webui/pipelines#383] [Q&A]
I used the function "Perplexity Search Manifold Pipe" for internet search, and it worked well until days ago. Now it shows "Error: Failed to get socket ID". May I ask if anyone can help? Thank you in advance.
![image](https://github.com/user-attachments/assets/4c8b63f4-6be9-4621-8678-90ee55f50b98)

**Comment by esni** [comment]
Same here, managed to solve it?
